include/micm/cuda/solver/cuda_solver_builder.hpp

```diff
@@ -21,7 +21,7 @@ namespace micm
   ///
   /// GPU solvers only work with vector-ordered matrices
   template<class SolverParametersPolicy, std::size_t L = MICM_DEFAULT_VECTOR_SIZE>
-  using CudaSolverBuilder = SolverBuilder<
+  using CudaSolverBuilder_Doolittle = SolverBuilder<
```

Comment:
```suggestion
  using DefaultCUDASolverBuilder = SolverBuilder<
```
I prefer this instead of adding doolittle to the name. What do you think?